src/Ethdebugger.js

```diff
@@ -46,9 +51,13 @@ Ethdebugger.prototype.render = function () {
         <div style='display:inline-block'>
           ${this.sticker.render()}
         </div>
+        <div style=${ui.formatCss(style.statusMessage)} >${this.statusMessage}</div>
         ${this.vmDebugger.render()}
      </div>`
-  )
+  if (!this.view) {
```

Comment:
We keep a reference of the return value of the first call to the yo'...' function . because this ref has to be used when we update the view: yo.update(this.view, this.render())